core/variant.cpp

```diff
@@ -513,6 +513,7 @@ bool Variant::can_convert_strict(Variant::Type p_type_from, Variant::Type p_type
 
 			static const Type valid[] = {
 				QUAT,
+				VECTOR3,
```

Comment:
I'm not sure if this change is needed. I understand that the type must be added to `Variant::can_convert`, but what about `Variant::can_convert_strict`? @reduz